docs/rules/no-warning-comments.md

```diff
@@ -1,33 +1,27 @@
 # Disallow Warning Comments (no-warning-comments)
 
-Often code is marked during development process for later work on it or with additional thoughts. Examples are typically `// TODO: do something` or `// FIXME: this is not a good idea`. These comments are a warning signal, that there is something not production ready in your code. Most likely you want to fix it or remove the comments before you roll out your code with a good feeling.
+Developers often add comments to code which is not complete or needs review. Most likely you want to fix or review the code, and then remove the comment before you roll out your code with a good feeling.
 
-## Rule Details
+```js
+// TODO: do something
+// FIXME: this is not a good idea
+```
 
-This rule can be used to help finding these `warning-comments`. It can be configured with an array of terms you don't want to exist in your code. It will raise a warning when one or more of the configured `warning-comments` are present in the checked files.
+## Rule Details
 
-The default configuration looks like this:
+This rule reports comments that warn something is not production ready in your code.
 
-```json
-"no-warning-comments": [0, { "terms": ["todo", "fixme", "xxx"], "location": "start" }]
-```
-
-This preconfigures
+## Options
 
-* the rule is disabled because it is set to `0`. Changing this to `1` for warn or `2` for error mode activates it (this works exactly the same as everywhere else in `ESLint`).
-* the `terms` array is set to `todo`, `fixme` and `xxx` as `warning-comments`. `terms` has to be an array. It can hold any terms you might want to warn about in your comments - they do not have to be single words. E.g. `really bad idea` is as valid as `attention`.
-* the `terms` are case-insensitive and are matched as whole words. E.g. `fix` would not match `fixing`.
-* the `location`-option set to `start` configures the rule to check only the start of comments. E.g. `// TODO` would be matched, `// This is a TODO` would not. You can change this to `anywhere` to check your complete comments.
+This rule has an options object literal:
 
-As already seen above, the configuration is quite simple. Example that enables the rule and configures it to check the complete comment, not only the start:
+* `terms`: optional array of terms to match. Defaults to `["todo", "fixme", "xxx"]`. Terms are matched case-insensitive and as whole words: `fix` would match `FIX` but not `fixing`. Terms can consist of multiple words: `really bad idea`.
```

Comment:
Yes, I choked here: double-minded because it is a rare example of two options, and lazy not to add simple example code for the default options.

A process question: I used :+1: reactions in 2 other line comments for practice, but as the person acting on your feedback, should I keep putting a thumbs up in a new comment so you know I will make the change and a committer can quickly see that all comments have been dealt with?